.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Scala CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '1.8.0', '11.0.x' ]
+        scala: [ '2.12.10' ]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: print Java version
+      run: java -version
+    - name: Run tests
+      run:  sbt ++${{ matrix.scala }} "^ test"
```

Comment:
I can't find documentation about what the `^` is in sbt command syntax, what does it do?

Reply:
the ^ character (Plugin Cross Command) is defined here:

https://github.com/sbt/sbt/blob/develop/main/src/main/scala/sbt/internal/CommandStrings.scala#L405

Cross build an SBT plugin:

  https://www.scala-sbt.org/1.x/docs/Cross-Build-Plugins.html

Some plugin authors declare this setting:

```
crossSbtVersions := Vector("1.2.8", "0.13.18")
```

If the plugin author does not declare crossSbtVersions, it defaults to the current SBT version.

Reply:
Ok. Since 0.13.x isn't compatible since scala.meta doesn't work for 2.10, this is just future compatibility stuff?